make deb fails on arm64

**Output of `docker version`:**

```
Client:
 Version:      1.10.3
 API version:  1.22
 Go version:   go1.6.1
 Git commit:   20f81dd
 Built:        Wed, 20 Apr 2016 14:19:16 -0700
 OS/Arch:      linux/arm64

Server:
 Version:      1.10.3
 API version:  1.22
 Go version:   go1.6.1
 Git commit:   20f81dd
 Built:        Wed, 20 Apr 2016 14:19:16 -0700
 OS/Arch:      linux/arm64
```


**Output of `docker info`:**

```
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 42
Server Version: 1.10.3
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 31
 Dirperm1 Supported: true
Execution Driver: native-0.2
Logging Driver: json-file
Plugins: 
 Volume: local
 Network: bridge null host
Kernel Version: 3.10.101-0-pine64-longsleep
Operating System: Ubuntu 16.04 LTS
OSType: linux
Architecture: aarch64
CPUs: 4
Total Memory: 1.943 GiB
Name: pine64
ID: NANA:36TS:63E4:JM3T:QQEE:CTEP:F324:WJHS:LXOL:7GNL:WNEN:JKKU
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Physical Pine64 with ubuntu 16.4 for arm64

**Steps to reproduce the issue:**
1. download git docker source
2. execute make deb

**Describe the results you received:**

---> Making bundle: dynbinary (in bundles/1.12.0-rc3/dynbinary)
Building: bundles/1.12.0-rc3/dynbinary-client/docker-1.12.0-rc3
github.com/miekg/pkcs11
vendor/src/github.com/miekg/pkcs11/pkcs11.go:26:18: fatal error: ltdl.h: No such file or directory
compilation terminated.
Makefile:86: recipe for target 'deb' failed
make: *** [deb] Error 1

**Describe the results you expected:**
deb docker package for installation

**Additional information you deem important (e.g. issue happens only occasionally):**
I allready installed libltdl-dev package and even tried to copy the file locally to pkcs11.go dir

I just tested, took a while but it worked perfectly!! Thanks, I didn't realise that the dependencies where needed on Dockerfile, I was stupidly trying to put them on the host.

Now it stacks on apparmor stuff (I append the error and I'll try to explore a bit) but seems to me that your pull request solves ltdl.h issue

```
---> Making bundle: .detect-daemon-osarch (in bundles/1.12.0-dev/build-deb)
find: `contrib/builder/deb/aarch64/*/': No such file or directory
---> Making bundle: .integration-daemon-stop (in bundles/1.12.0-dev/build-deb)
+++++ cat bundles/1.12.0-dev/build-deb/docker.pid
++++ kill 5063
++++ /etc/init.d/apparmor stop
 * Clearing AppArmor profiles cache
   ...done.
All profile caches have been cleared, but no profiles have been unloaded.
Unloading profiles will leave already running processes permanently
unconfined, which can lead to unexpected situations.

To set a process to complain mode, use the command line tool
'aa-complain'. To really tear down all profiles, run the init script
with the 'teardown' option."
Makefile:86: recipe for target 'deb' failed
make: *** [deb] Error 1
```

--- a/Dockerfile.aarch64
+++ b/Dockerfile.aarch64
@@ -36,6 +36,7 @@ RUN apt-get update && apt-get install -y \
 	libapparmor-dev \
 	libc6-dev \
 	libcap-dev \
+	libltdl-dev \
 	libsqlite3-dev \
 	libsystemd-dev \
 	mercurial \

